packages/combobox/package.json

```diff
@@ -5,14 +5,14 @@
 	"module": "dist/ag.ds-next-combobox.esm.js",
 	"license": "MIT",
 	"peerDependencies": {
-		"@ag.ds-next/box": "6.0.0",
-		"@ag.ds-next/button": "9.1.0",
-		"@ag.ds-next/core": "3.0.0",
-		"@ag.ds-next/field": "8.0.0",
-		"@ag.ds-next/icon": "9.0.0",
-		"@ag.ds-next/loading": "4.0.0",
-		"@ag.ds-next/text": "8.0.0",
-		"@ag.ds-next/text-input": "9.0.0",
+		"@ag.ds-next/box": "7.0.0",
+		"@ag.ds-next/button": "11.0.0",
+		"@ag.ds-next/core": "4.0.0",
+		"@ag.ds-next/field": "10.0.0",
+		"@ag.ds-next/icon": "11.0.0",
+		"@ag.ds-next/loading": "6.0.0",
+		"@ag.ds-next/text": "10.0.0",
+		"@ag.ds-next/text-input": "11.0.0",
```

Comment:
I had to update these dependencies manually.
`yarn manypkg fix` doesn't seem to know how to fix this for us here, since we're using `"*"` as the version in the `devDependencies`.

Reply:
Whoa... how did that happen 🤔